src/fragments/lib/project-setup/ios/upgrade-guide/upgrade-guide.mdx

```diff
@@ -1,11 +1,9 @@
-As of v2, Amplify Library iOS is now Amplify Library for Swift. Amplify Library for Swift supports iOS 13+ and macOS 10.15+, and ships with APIs that leverage Swift Concurrency (async/await) to return values. There are a few things you need to know if you are upgrading to this latest version.
+As of v2, Amplify Library iOS is now Amplify Library for Swift. Amplify Library for Swift supports iOS 13+, macOS 10.15+, tvOS 13+, watchOS 9+, and visionOS (developer preview), and adopts Swift structured concurrency including async/await. There are a few things you need to know if you are upgrading to this latest version.
```

Comment:
tvOS, watchOS, and visionOS (preview) support have been added to Amplify Swift v2 since it was released